src/pages/thanks-page/order-data/order-data.js

```diff
@@ -1,30 +1,28 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import { Table, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faDollarSign, faHryvnia } from '@fortawesome/free-solid-svg-icons';
 
-import { CART_TABLE_FIELDS } from '../../../translations/cart.translations';
 import { useStyles } from './order-data.styles';
-import { PAYMENT_STATUS, THANKS_PAGE_TITLE } from '../../../translations/thanks-page.translations';
 import OrderItemCard from './order-item-card';
 import { defaultProps, ORDER_PAYMENT_STATUS, thanksPropTypes } from '../../../utils/thank-you';
 
 const OrderData = ({ language, currency, order, isLightTheme }) => {
```

Comment:
After refactoring you don't need 'language' in this file any more

Reply:
changed